pkgs/applications/version-management/git-and-tools/git-extras/default.nix

```diff
@@ -1,16 +1,22 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, unixtools, which }:
 
 stdenv.mkDerivation rec {
   pname = "git-extras";
-  version = "4.7.0";
+  version = "5.0.0";
 
   src = fetchurl {
     url = "https://github.com/tj/git-extras/archive/${version}.tar.gz";
```

Comment:
GitHub _archives_ are regenerated from time to time and the hash may change. You should rather use `fetchzip` or `fetchFromGitHub` here.
See #32997 for more context.

Reply:
fixed